amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -1,22 +1,32 @@
 //! Local transform component.
 
-use cgmath::{Matrix3, Matrix4, Quaternion, Vector3};
+use cgmath::{Angle, Deg, Euler, InnerSpace, Matrix3, Matrix4, Point3, Quaternion, Rotation, Rotation3, Vector3};
+use orientation::Orientation;
 use specs::{Component, DenseVecStorage, FlaggedStorage};
 
 /// Local position, rotation, and scale (from parent if it exists).
 ///
 /// Used for rendering position and orientation.
 #[derive(Clone, Debug, PartialEq, PartialOrd)]
 pub struct LocalTransform {
-    /// Translation/position vector [x, y, z]
-    pub translation: [f32; 3],
     /// Quaternion [w (scalar), x, y, z]
     pub rotation: [f32; 4],
     /// Scale vector [x, y, z]
     pub scale: [f32; 3],
+    /// Translation/position vector [x, y, z]
+    pub translation: [f32; 3],
 }
 
 impl LocalTransform {
+    /// Rotate to look at a point in space (without rolling)
+    pub fn look_at(&mut self, orientation: &Orientation, position: Vector3<f32>) -> &mut Self {
+        let cam_quat = Quaternion::from(self.rotation);
```

Comment:
Unused variable

Reply:
wow, not even the compiler displays a warning about that one, but true, it's unused!

Reply:
It did here: https://travis-ci.org/amethyst/amethyst/jobs/291734426#L806

Reply:
strange, my local compiler didn't show that one (only a warning in `amethyst_renderer\src\formats\texture.rs:219:5`)~

Reply:
Run with nightly compiler.